### PR TITLE
Added A New Book - quantum (subject : Physics)

### PR DESCRIPTION
**Pull request by MachO7**
## What does this PR do?
Add Resource-- Adds another book

### Checklist:
- [check ] Not a duplicate
- [check ] Included author(s) if appropriate
- [Check ] Lists are in alphabetical order
- [Check ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-science-books.md [hunk 1]
```diff
@@ -80,33 +80,65 @@
...
+
 * [Motion Mountain Physics](http://www.motionmountain.net) - Christoph Schiller
+
 * [Physics Before and After Einstein](http://www.dmi.unipg.it/mamone/pubb/PBAE.pdf) - Marco Mamone Capria (PDF)
+
 * [Physics Test Practice Book](https://www.physics.ohio-state.edu/undergrad/greStuff/exam_GR9677.pdf) - (PDF)
+
 * [Problems in Introductory Physics](http://www.lightandmatter.com/problems/problems.pdf) - B. Crowell and B. Shotwell (PDF)
+
+* [Quantum : Einstein,Bohr and the great debate about the nature of reality](https://www.pdfdrive.com/quantum-einstein-bohr-and-the-great-debate-about-the-nature-of-reality-e193612567.html) - Manjit Kumar (PDF)
```
Comment (eshellman) [inline free-science-books.md:122]: In general, we do not link to file sharing sites.

**Review by eshellman — CHANGES_REQUESTED** [review]
good list (except for one) but need to clean up the spaces to satisfy linter.

**Comment by MachO7** [comment]
Thank You,
So, I need to align the spaces and we'll be good?

**Comment by eshellman** [comment]
right

**Comment by MachO7** [comment]
I have made the necessary changes. The Space issues are sorted, the link now directs to Mega instead of PDFDrive since you said that file sharing sites aren't appreciated... Thanks

**Comment by MachO7** [comment]
Hello Eric,
Thanks for helping me out,
I have made the necessary changes
I have also commented about the same
in the pull-request I had made on GitHub.
I hope my PR is accepted now.
Regards,
Ojas,

**Comment by eshellman** [comment]
mega is a file sharing site

**Pull request closed without merging**